### ISSUE TITLE
Gr00t N1.5

### ISSUE THREAD

**Issue by andrearosasco** [andrearosasco/papers#27]
#### Architecture
The main differences from N1 are the following:
- The VLM model is frozen during both pretraining and finetuning.
- The adapter MLP connecting the vision encoder to the LLM is simplified and adds layer normalization to both visual and text token embeddings input to the LLM.
We found that these modifications greatly improved language following and generalization.

Other modifications:
- Using FLARE instead of LAPA
- Changed neural trajectory generation to DreamGen
- New backbone, Eagle 2.5 is tuned for better grounding capabilities and physical understanding.
